ci/docker/android-emulator/hardware/config_29.ini

```diff
@@ -23,7 +23,7 @@ hw.sensors.pressure=no
 hw.sensors.humidity=no
 hw.sensors.magnetic_field_uncalibrated=no
 hw.sensors.gyroscope_uncalibrated=no
-image.sysdir.1=system-images/android-29/google_apis/x86_64/
+image.sysdir.1=system-images/android-29/google_apis/x86/
```

Comment:
If I'm not mistaken we have an implicit contract with reservation config.
See how we use `model` (`Android_SDK_built_for_x86_64`) from reservation config in `IntentionsRouter`.
Didn't find an issue for it. If so, we need to make at least fail fast in this case.

Reply:
you're right, let me fix it

Reply:
reverted arch change